init.lua

```diff
@@ -166,6 +166,15 @@ vim.o.scrolloff = 10
 -- See `:help 'confirm'`
 vim.o.confirm = true
 
+-- Enable undo/redo changes even after closing and reopening a file
+vim.o.undofile = true
```

Comment:
This already exists at [line 125](https://github.com/nvim-lua/kickstart.nvim/pull/1642/files#diff-0fc5c43f6f1d7605fd7d16c088ba6b83f096eee8e2049517407bbaa27aa3a4f7L125)